docs/usage.md

```diff
@@ -370,46 +371,49 @@ More informations are available in the [Sourcepredict documentation](https://sou
 
 ### `--name3`
 
-Name of candidate 1. Example: "Sus_scrofa"
+Name of candidate species 3.
 
-### `--fasta3`
```

Comment:
You go up to three now!? cool! Should also be described somewhere in a  bit more detail this functionality exists and what it does

Reply:
Same as above, description added in README a6ddc48